app/configurator/interactive-filters/time-slider.tsx

```diff
@@ -75,17 +75,19 @@ export const TimeSlider = (props: TimeSliderProps) => {
   const componentFilter = filters[component.iri];
 
   const timelineProps: TimelineProps = React.useMemo(() => {
-    console.log("timelineProps");
     const commonProps = {
       animationType,
       msDuration: animationDuration * 1000,
     };
-    const values = chartState.allData
-      .map((d) => d[component.iri])
-      .filter(Boolean) as string[];
+    const uniqueValues = Array.from(
+      new Set(
+        chartState.allData
+          .map((d) => d[component.iri])
+          .filter(Boolean) as string[]
```

Comment:
I think there is a isTruthy or a truthy helper that can replace the Boolean here, and would help not having to do the `as string[]`.